cmd/show.go

```diff
@@ -66,10 +67,13 @@ var showCmd = &cobra.Command{
 		}
 
 		objs, err := expandEnvCmdObjs(cmd, envSpec, wd)
+		out := cmd.OutOrStdout()
```

Comment:
Does this need to be pulled into an a variable? I don't think we're using it outside of `c.Run`. (Also it seems especially odd to put it between the declaration of `err` and `if err != nil`.)

Reply:
whoops ... I was wanting to use `out` for the outputting earlier instead of `log` but forgot to change back ..